src/Features/header/components/CityLanguageSelect/CityLanguageSelect.tsx

```diff
@@ -0,0 +1,12 @@
+import { ReactElement } from 'react';
+import { CitySelect } from './City/CitySelect';
+import { Language } from './Language';
+
+export const CityLanguageSelect = (): ReactElement => {
+  return (
+    <div className="hero__city-select flex flex-start absolute top-0 right-0 align-items-start align-top">
+      <Language languageValue={undefined} />
```

Comment:
See, this passing of `undefined` is very inconvenient, we can do better with optional member of interface/type